js/calc.js

```diff
@@ -0,0 +1,87 @@
+// var buttonClear = document.getElementById('1');
+// buttonClear.addEventListener('click', function(event){
+//   document.getElementById('screen').innerHTML = "1"
+// })
+
+// for loop for the individual buttons
+// numbered 1-9
+function Numbers(){
+var buttons = document.querySelectorAll(".buttonNum");
+
+for (var i = 0; i< buttons.length; i++) {
+  var button = buttons[i];
+
```

Comment:
buttons is an array; you can't store them in one variable and then add an event listener to all of them like this. You would need to loop through the array and add it to each one.